app/models/greeting.rb

```diff
@@ -1,3 +1,4 @@
 class Greeting < ApplicationRecord
-  # frozen_string_literal: true
+  validates :name, presence: true
```

Comment:
- [ ] There is no name field in the greetings table. I notice you are validating for the presence of a name while there is no name in the greetings table. Kindly remove this please as it is returning an error.